release-highlights/release-highlights-2021.01/release-highlights.md

```diff
@@ -0,0 +1,62 @@
+
+### [Code editor](https://www.patternfly.org/v4/components/code-editor)
+
+![code editor](./img/code-editor.png)
+
+ Introduce native support for the Monaco code editor to format code for read-only display or for editing. Includes language appropriate syntax highlighting, a dark theme option, customizable actions.
+
+ ### [Slider component](https://www.patternfly.org/v4/components/slider)
+
+![slider examples](./img/slider.png)
+
+ The Slider provides basic slider capability for entering numeric data. Both discrete and continuous sliders are supported. An optional value input field supports either direct entry from the keyboard or manipulation of the slider using the mouse or keyboard.
+
+### [Number input](https://www.patternfly.org/v4/components/number-input)
+
+![number input component](./img/number-input.png)
+
+The Touchspin component has been renamed to Number input. This component is designed for easily entering numeric data.
+
+-----------------------------------------------------------------------------
+
+## What we’re working on...
+
+### 2021.02 (February 19)
+
+* [Multi-level drill down menu](https://github.com/patternfly/patternfly-react/issues/5024) - introduces a drill-down menu component for displaying hierarchical data.
+
+* [Attribute-value search input](https://github.com/patternfly/patternfly-design/issues/771) - add a new search input variant to support advanced searching or filtering use cases that require an attribute-value pair.
+
+* [Updated Label styling](https://github.com/patternfly/patternfly/issues/3810) - Update label styles to have better visibility against any background.
+
+* [Log viewer component](https://github.com/patternfly/patternfly-react/issues/5341) - introduce a native log viewer native beta component to display real-time streamed logs or static log data.
+
+* [Large/Marketing card variant](https://github.com/patternfly/patternfly/issues/3670) - introduce a card with larger spacing for marketing applications.
+
+* [SVG Spinner](https://github.com/patternfly/patternfly-react/issues/5343) - introduce a new SVG spinner variant for smoother rendering and performance.
+
+### 2021.03 (March 12)
+
+* [Jump links demo](https://github.com/patternfly/patternfly/issues/3560) - illustrate how jump links can be used in a sticky sidebar.
+
+* [New List variants](https://github.com/patternfly/patternfly-react/issues/5336) - Introduce new variants of the [List](https://www.patternfly.org/v4/components/list) component that include plain lists, lists with horizontal separators, and lists with icons for communicating static information in a standard `<li>`element.
+
+* [Expandable form sections](https://github.com/patternfly/patternfly/issues/3557) - these will be expandable nested sections useful for organizing information on long, complex forms.
```

Comment:
It looks like all other links point to open issues, but this links to a closed issue - should we link to [the open React issue](https://github.com/patternfly/patternfly-react/issues/5023) instead?